.github/workflows/actions.yml

```diff
@@ -12,7 +12,9 @@ jobs:
       - name: Checkout benchmarks
         uses: actions/checkout@v2
       - name: Setup Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
```

Comment:
necessary because pyarrow doesn't have 3.11 wheels yet, and building from pyarrow.tar.gz is complicated